superset-websocket/client-ws-app/public/javascripts/app.js

```diff
@@ -0,0 +1,42 @@
+console.log('hello');
+let socketCount = 0;
+let messageCount = 0;
+
+const jwtSecret = "test-secret-change-me";
```

Comment:
how are you planning to do config management here? I assume this will need to be the same secret as in the flask app? Perhaps we should use an environment var here

Reply:
This should read from the same config file as the main app, which still needs to be moved out of version control. The `client-ws-app` is not meant to be used in production, only for testing; I'll improve the packaging and documentation to make that clear.